tests/vasp/test_vasp.py

```diff
@@ -67,7 +67,7 @@ def test_potential(self):
         self.assertEqual(self.job.potential, self.job._potential)
 
     def test_plane_wave_cutoff(self):
-        self.assertIsInstance(self.job.plane_wave_cutoff, (float, int))
+        # self.assertIsInstance(self.job.plane_wave_cutoff, (float, int))
```

Comment:
I think we should keep this line. Just use

`self.assertIsInstance(self.job.plane_wave_cutoff, (float, int, type(None))`

Reply:
My assumption would be that the tests fail with those changes, but let's give it a try.

Reply:
I have to correct myself - it worked, just missing a `)`